Lib/crypt.py

```diff
@@ -45,7 +45,19 @@ def mksalt(method=None, *, rounds=None):
     else:  # modular
         s = f'${method.ident}$'
 
-    if method.ident and method.ident[0] == '2':  # Blowfish variants
+    if method.ident and method.ident == 'y':  # yescrypt
+        if rounds is not None:
+            if not 1 <= rounds <= 11:
+                raise ValueError('rounds out of the range 1 to 11')
+        else:
+            rounds = 5
+        if rounds < 3:
+            s += 'j' + chr(54 + rounds) + '5$'
+        elif rounds < 6:
+            s += 'j' + chr(52 + rounds) + 'T$'
+        elif rounds < 12:
+            s += 'j' + chr(59 + rounds) + 'T$'
```

Comment:
Would not be easier to write it as
```suggestion
        s += 'j' + ('75', '85', '7T', '8T', '9T', 'AT', 'BT', 'CT', 'DT', 'ET', 'FT')[rounds - 1] + '$'
```

Reply:
Thank you! Looks more pythonic indeed.